.env.example

```diff
@@ -0,0 +1,8 @@
+SECRET_KEY_BASE=
+BASE_URL=
+DATABASE_USER=
+DATABASE_PASSWORD=
+POSTGRES_MULTIPLE_DATABASES=project_tides_db,project_tides_test_db
+DATABASE_DB=project_tides_db
+DATABASE_HOST=db
+DATABASE_ROLE=project_tides_db #As per the db_init.sh script, role and db name are same
```

Comment:
what is database role? If this is same as db name, why do we need a different variable to store it?

Reply:
In Postgres, the role is basically like a super admin that has all the privileges except login.
By default, it is `postgres` but we have updated based on the database we are creating.
The name seems confusing, I'm updating it to `POSTGRES_ROLE` as it will not be used for non-postgres users.